File: Proptimo/Backend/Proptimo/Core/Proptimo.Domain/Entities/RealEstateAddress.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Proptimo.Domain.Entities
{
    public class RealEstateAddress
    {
        public string RealEstateAddressId { get; set; }
        public string CityName { get; set; }           
        public string DistrictName { get; set; }       
        public string NeighborhoodName { get; set; }  
        public string? Street { get; set; }            
        public string? BuildingNo { get; set; }
        public string DoorNumber { get; set; }
        public decimal Latitude { get; set; }          
        public decimal Longitude { get; set; }         
     
    }
}

File: Proptimo/Frontend/proptimo/src/features/api/types/realEstateAddress.ts
export interface RealEstateAddress {
 id: string,
 cityName: string,
 districtName: string,
 neighborhoodName: string,
 street: string,
 buildingNo: string,
 doorNumber: string,
 latitude: number,
 longitude: number,
 realEstateId: string
}

export interface CreateRealEstateAddress {
    cityName: string,
    districtName: string,
    neighborhoodName: string,
    street: string,
    buildingNo: string,
    doorNumber: string,
    latitude: number,
    longitude: number,
    realEstateId: string
}

export interface GetAddressByEstateId {
    id: string,
    cityName: string,
    districtName: string,
    neighborhoodName: string,
    street: string,
    buildingNo: string,
    doorNumber: string,
    latitude: number,
    longitude: number,
    realEstateId: string
}



export interface UpdateRealEstateAddress {
    id: string,
    cityName: string,
    districtName: string,
    neighborhood: string,
    street: string,
    buildingNo: string,
    doorNumber: string,
    latitude: number,
    longitude: number,
    realEstateId: string
}